prime-dotnet-webapi/Services/OrganizationService.cs

```diff
@@ -78,15 +78,19 @@ public async Task<IEnumerable<OrganizationListViewModel>> GetOrganizationsByPart
                 .Include(o => o.SigningAuthority)
                         .ThenInclude(sa => sa.Addresses)
                             .ThenInclude(pa => pa.Address)
+                // .Include(o => o.Sites)
+                //     .ThenInclude(s => s.SiteStatuses)
```

Comment:
Commented out code?

Reply:
Yeah, we should talk about `DecompileAsync` again and then I'll remove